### DISCUSSION TITLE
Dropdown Menu Open on Hover

### DISCUSSION THREAD

**Discussion by jeffpohlmeyer** [huntabyte/shadcn-svelte#196] [Help]
### Describe the feature in detail (code, mocks, or screenshots encouraged)

Trying to port over a Vue/Vuetify-based project and one big helper would be the ability to open a menu upon hover. Looking at something like this: https://vuetifyjs.com/en/components/menus/#open-on-hover

It seems that the [tooltip](https://www.shadcn-svelte.com/docs/components/tooltip) has open-on-hover functionality and you can have that mimic the sort of thing I'm looking for (and Melt-UI includes positioning) but I'm wondering that's the way I should be going for a hoverable menu or a Dropdown Menu would be the appropriate course of action.

### What type of pull request would this be?

Enhancement

**Comment by huntabyte** [comment]
This would be something I'd have to make possible from the [bits-ui](https://github.com/huntabyte/bits) side as I'd have to forward the proper events to make this possible, but it'd require some custom logic on your end to make it work as expected.

**Comment by voidrlm** [comment]
Just use the  mouseover or mouseleave event by Vue and programmatically open the drop down menu list.

Example:
@mouseover="hover = true"
@mouseleave="hover = false"

**Reply by jeffpohlmeyer** [reply]
Not quite. If you do that then you don't have any interactivity with the menu that opens up because as soon as you hover into the menu you're no longer hovering over the trigger/activator and the menu will close

**Reply by JustMrMendez** [reply]
Any updates on this? i'm trying to recreate the navigation menu from shadcn and i'm unsure what would be the best way of doing this
![image](https://github.com/huntabyte/shadcn-svelte/assets/56850299/c799f2eb-3943-48bc-a373-d283bc30cb29)
